src/test/groovy/it/move2/agent/application/JobFixture.groovy

```diff
@@ -0,0 +1,26 @@
+package it.move2.agent.application
+
+import it.move2.job.EmploymentType
+import it.move2.job.JobOffer
+import it.move2.job.Skill
+
+class JobFixture {
+
+    static jobOffer(args = [:]) {
+        new JobOffer(
+                'id' in args.keySet() ? args.id as String : 'test-id',
+                'title' in args.keySet() ? args.title as String : 'test-title',
+                'companyName' in args.keySet() ? args.companyName as String : 'test-company',
+                'experience' in args.keySet() ? args.experience as String : 'test-experience',
+                'skills' in args.keySet() ? args.skills as List<Skill> : [],
+                'employmentType' in args.keySet() ? args.employmentType as List<EmploymentType> : [],
+                'remote' in args.keySet() ? args.remote as boolean : true
+        )
+    }
+
+    static List<JobOffer> createListJobOffers(int count) {
```

Comment:
The name of the function "createListJobOffers" may not reflect the intent well, as it may be associated with creating a list and storing it in a database, which is not the case here. I propose the same naming convention as above ie: jobOffersList